sites/fast-website/src/app/components/side-navigation/side-navigation.styles.ts

```diff
@@ -2,56 +2,54 @@ import { css } from "@microsoft/fast-element";
 import { display } from "@microsoft/fast-foundation";
 import {
     accentForegroundActiveBehavior,
+    accentForegroundCutRestBehavior,
     accentForegroundHoverBehavior,
-    neutralForegroundRestBehavior,
 } from "@microsoft/fast-components";
 
 export const SideNavigationStyles = css`
-    ${display("flex")}:host {
-        --engaged-height: calc(var(--design-unit) * 9px);
+    ${display("flex")} :host {
+        --active-height: calc(var(--design-unit) * 13px);
         --design-unit-large: calc(var(--design-unit) * 5px);
         contain: content;
         font-family: var(--body-font);
-        flex-direction: column;
-        justify-content: center;
     }
 
-    ul {
+    .item-list {
         list-style-type: none;
+        margin: 0;
         padding: 0 var(--design-unit-large);
-        cursor: pointer;
     }
 
-    li {
-        height: calc(var(--design-unit) * 12px);
+    .list-item {
         display: flex;
         flex-direction: column;
         justify-content: center;
     }
 
-    li:hover .icon {
+    .icon {
+        fill: var(--accent-foreground-cut-rest);
+    }
+
+    .list-item:hover .icon {
         fill: var(--accent-fill-hover);
     }
 
     .scroll-indicator {
-        height: var(--design-unit-large);
-        width: calc(var(--design-unit) * 1px);
-        background-color: ${neutralForegroundRestBehavior.var};
+        height: calc(var(--design-unit) * 9px);
```

Comment:
Similar question to above - does this change with the design unit, or should it just be hardcoded?